src/harness/security_testcase.py

```diff
@@ -269,6 +269,7 @@ def assertTlsHandshakeFailureOrHttp403(self, client_cert=None, client_key=None,
     try:
       self.assertTlsHandshakeFailure(client_cert, client_key, ciphers, ssl_method)
```

Comment:
You have two choise:
1) change 
     client_cert = client_cert or sas.GetDefaultCbsdSSLCertPath()
     client_key = client_key or sas.GetDefaultCbsdSSLKeyPath()
in    def assertTlsHandshakeFailure(self, client_cert=None, client_key=None, ciphers=None, ssl_method=None):

to
     client_cert = client_cert or self.getCertFilename('device_a.cert')
     client_key = client_key or self.getCertFilename('device_a.key')
or keeping sas.GetDefaultCbsdSSLCertPath() and sas.GetDefaultCbsdSSLKeyPath()   and load device_a.cert and device_a.key for those function